axiom/monitors.go

```diff
@@ -67,15 +82,37 @@ func (c *Operator) UnmarshalJSON(b []byte) (err error) {
 	return err
 }
 
+// MarshalJSON implements [json.Marshaler]. It is in place to marshal the
+// Type to its string representation because that's what the server
+// expects.
+func (c Type) MarshalJSON() ([]byte, error) {
+	return json.Marshal(c.String())
+}
+
+// UnmarshalJSON implements [json.Unmarshaler]. It is in place to unmarshal the
+// Type from the string representation the server returns.
+func (c *Type) UnmarshalJSON(b []byte) (err error) {
+	var s string
+	if err = json.Unmarshal(b, &s); err != nil {
+		return err
+	}
+
+	*c = typeFromString(s)
+
+	return nil
+}
+
 type Monitor struct {
 	// ID is the unique ID of the monitor.
 	ID string `json:"id,omitempty"`
+	// Type sets the type of monitor. Defaults to Threshold
+	Type Type `json:"type"`
 	// AlertOnNoData indicates whether to alert on no data.
 	AlertOnNoData bool `json:"alertOnNoData"`
 	// NotifyByGroup tracks each none-time group independently
 	NotifyByGroup bool `json:"notifyByGroup"`
 	// Resolvable determines whether the events triggered by the monitor
-	// are resolvable. This has no effect on threshold monitors
+	// are resolvable. This has no affect on threshold monitors
```

Comment:
nit: was this intended?

Reply:
It was intended as I figured it was being used as a verb, but turns out, no.